src/handlers/read-storage-handler.js

```diff
@@ -24,7 +24,7 @@ const _noOp = () => undefined
 export function StorageHandler (storageStrategy, externalStorageHandler) {
   const errors = []
   function _externalOrError (functionName) {
-    const hasExternal = externalStorageHandler && externalStorageHandler[functionName] && isFunction(externalStorageHandler[functionName])
+    const hasExternal = !strEqualsIgnoreCase(storageStrategy, StorageStrategy.disabled) && externalStorageHandler && externalStorageHandler[functionName] && isFunction(externalStorageHandler[functionName])
```

Comment:
We will send an errors in line 31, even though we decided to go with _noOp because of the disabled strategy.

Reply:
Also, when sending errors we need to make sure, that we also send the privacy related parameters. Currently, the privacy related flags are appended after the errors callback was registered.